js/renovation/ui/scroll_view/__tests__/topPocket.test.tsx

```diff
@@ -46,4 +50,29 @@ describe('TopPocket', () => {
       });
     });
   });
+
+  describe('Behavior', () => {
+    each([
+      { state: undefined, expectedText: 'pullingDownText' },
+      { state: TopPocketState.STATE_RELEASED, expectedText: 'pullingDownText' },
+      { state: TopPocketState.STATE_READY, expectedText: 'pulledDownText' },
+      { state: TopPocketState.STATE_REFRESHING, expectedText: 'refreshingText' },
+    ]).describe('State: %o', (testConfig) => {
+      it('Correct text is visible depending of state', () => {
+        const viewModel = new TopPocket({
+          refreshStrategy: 'simulated',
+          pullingDownText: 'pullingDownText',
+          pulledDownText: 'pulledDownText',
+          refreshingText: 'refreshingText',
+        });
+        if (testConfig.state) {
+          // eslint-disable-next-line no-underscore-dangle
+          viewModel._state = testConfig.state;
+        }
+        const topPocket = mount(viewFunction(viewModel) as JSX.Element);
+        const textElement = topPocket.find('.dx-scrollview-pull-down-text-visible');
+        expect(textElement.text()).toBe(testConfig.expectedText);
```

Comment:
Check the count of elements with visible class. Should be single item only.

Reply:
Yep fixed